taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -91,22 +114,32 @@ def build_browsertime_task(config, tasks):
         task["run"]["command"].append("--test={}".format(test_name))
         task["run"]["command"].extend(task.pop("args", []))
 
+        # Setup treherder symbol
+        symbol = task.pop("subtest-symbol", None)
+
+        # taskcluster is merging task attributes with the default ones
+        # resulting the --cold extra option in the ytp warm tasks
+        if "youtube-playback" in task["name"]:
+            symbol = test_name.replace("youtube-playback-", "ytp-")
+
+        # Setup chimera for combined warm+cold testing
+        if task.pop("chimera", False):
+            task["run"]["command"].append("--chimera")
+
+        # Add '-c' to taskcluster symbol when running cold tests
+        elif "--cold" in task["run"]["command"]:
+            symbol += "-c"
+
         # Setup visual metrics
         run_visual_metrics = task.pop("run-visual-metrics", False)
         if run_visual_metrics:
             task["run"]["command"].append("--browsertime-video")
             task["run"]["command"].append("--browsertime-no-ffwindowrecorder")
             task["attributes"]["run-visual-metrics"] = True
 
-        # Setup chimera for combined warm+cold testing
-        if task.pop("chimera", False):
-            task["run"]["command"].append("--chimera")
-
-        # taskcluster is merging task attributes with the default ones
-        # resulting the --cold extra option in the ytp warm tasks
-        if 'youtube-playback' in task["name"]:
-            task["run"]["command"].remove("--cold")
-
+        # Build taskcluster group and symol
+        task["treeherder"]["symbol"] = "Btime(%s)" % symbol
+        task["name"] = task["name"].replace("tp6m-", "tp6m-{}-".format(symbol))
```

Comment:
I see that the test names have a `-hv` in them, can we get rid of that with another replace?